CRM/Utils/Mail.php

```diff
@@ -301,8 +302,13 @@ public static function send(&$params) {
     }
 
     if (is_object($mailer)) {
-      $errorScope = CRM_Core_TemporaryErrorScope::ignoreException();
-      $result = $mailer->send($to, $headers, $message);
+      try {
+        $result = $mailer->send($to, $headers, $message);
+      }
+      catch (Exception $e) {
+        CRM_Core_Session::setStatus($e->getMessage(), ts('Mailing Error'), 'error');
```

Comment:
is there a reason why your not logging this via errorMessage?

Reply:
It's consisent with a couple of lines lower down / what happened before

Reply:
Doh thought it was a logger np then